pages/api/v1/activity-feed/subscribed-posts.ts

```
@@ -0,0 +1,335 @@
+					const topic_id = data?.topic_id;
+
+					const post_reactionsQuerySnapshot = await postDocRef.collection('post_reactions').get();
+					const reactions = getReactions(post_reactionsQuerySnapshot);
+
+					const sentiments: { [key: number]: number } = {};
+					const commentsQueryDocs = await postDocRef.collection('comments').where('isDeleted', '==', false).get();
+
+					commentsQueryDocs.docs.map((doc) => {
+						if (doc.exists) {
+							const data = doc.data();
+							if (!isNaN(data?.sentiment)) {
```

Biome

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)